### DISCUSSION TITLE
Become a Fonoster Champion

### DISCUSSION THREAD

**Discussion by psanders** [fonoster/yaps#218] [General]
Let's start this conversation

**Comment by Igcorreia** [comment]
Hi @psanders interested :)

**Reply by psanders** [reply]
Hi @Igcorreia 

I'm putting together the program and hope to have the initial onboarding process for next. I will make sure you are the first to find out. Do you know anyone else who might be interested?

PS: I love your [Project](https://igcorreia.com/) page, it looks slick :)

**Reply by Igcorreia** [reply]
Hum not really, and probably too soon to share it, maybe when there is more info.
Was looking for something like this for a long time.

**Reply by psanders** [reply]
This is a top priority so you will get detailed information soon 🥁🥁

**Reply by psanders** [reply]
@Igcorreia 

I didn't forget about this, just been unusually busy. 
I'm curious, are you already in our Slack community?

**Reply by Igcorreia** [reply]
No not yet. :) Can I join?

**Reply by psanders** [reply]
For sure, here is the link: https://join.slack.com/t/fonoster/shared_invite/zt-b6n7hrwz-PJT9SXg_RRR9Q1gI7DI6kA

**Comment by AlijiEmmanuel-dev** [comment]
I really love this community and I will like to be a contributor.
I am currently working on some use cases with Fonoster, hopefully I will deploy before 1 quarter next year.

**Reply by psanders** [reply]
Hi @AlijiEmmanuel-dev 

Have you joined the [Discord](https://discord.gg/mpWSRUhG7e) channel? Looking forward to collaborating. 

cc: @YuriCodes
